service/src/main/java/com/ge/predix/acs/attribute/readers/ExternalAttributeReader.java

```diff
@@ -120,7 +120,7 @@ CachedAttributes getAttributesFromAdapters(final String identifier) throws Attri
             try {
                 attributesResponse = this.getAdapterOauth2RestTemplate(adapterConnection).
                         getForEntity(adapterUrl, AttributesResponse.class).getBody();
-            } catch (final Throwable e) {
+            } catch (final Exception e) {
```

Comment:
In general, thats true - however if we are able to do some error handling - its worth a try.  e.g. if there is a 'Error' thrown in evaluation service layer, its probably worth attempting to return a HTTP response with Effect.INDETERMINATE (instead of HTTP 500).  If jvm is out of memory its probably going to fail in the next step anyway.